google/cloud/edgenetwork/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 
 include(GoogleCloudCppLibrary)
 
-set(GOOGLE_CLOUD_CPP_SERVICE_DIRS "v1/")
+set(GOOGLE_CLOUD_CPP_SERVICE_DIRS "" "v1/")
```

Comment:
Why did we need to add `""`?

Reply:
For the mocks. You need it for the CI to pass. edgecompute is the same.

Reply:
Hmmm.  It sounds like we should not have set `forwarding_product_path: "google/cloud/edgenetwork"`.  It should only be used for backwards compatibility with a previous, non-versioned release of the library.  It may be worth regenerating if that is the case.